src/nepiada/env/nepiada.py

```diff
@@ -165,9 +172,10 @@ def step(self, actions):
 
         - infos
             Is a dictionary with agent_names as the key. Each value in turn is a dict
-            of the form {"obs": [], "comm": []}
+            of the form {"obs": [], "comm": [], "beliefs": []}
```

Comment:
We can probably get rid of "obs" from infos if we're just returning the observations.